examples/configFiles/deepMedic/train/trainConfig.cfg

```diff
@@ -11,6 +11,8 @@ folderForOutput = "../../../output/"
 #  [Optional] Path to a saved model, to load parameters from at beginning of the session. If one is also specified from command line, the latter will be used.
 #cnnModelFilePath = "../../../output/models/placeholder"
 
+#  [Optional] Log to Tensorboard, set to 0 or remove to suppress logging to tensorboard
+tensorboardLog = 1
```

Comment:
First, I think since this can only take two values, True or False, we may as well make them just give those. Instead of arbitrary numbers. Small difference, but checking.